package.json

```diff
@@ -99,25 +99,25 @@
     "react-dom": "^15.6.2 || ^16.2.0"
   },
   "sassIncludes": {
-    "patternfly": "--include-path node_modules/patternfly/dist/sass",
-    "bootstrap": "--include-path node_modules/patternfly/node_modules/bootstrap-sass/assets/stylesheets",
-    "fontAwesome": "--include-path node_modules/patternfly/node_modules/font-awesome-sass/assets/stylesheets"
+    "patternfly": "node_modules/patternfly/dist/sass",
+    "bootstrap": "node_modules/bootstrap-sass/assets/stylesheets",
+    "fontAwesome": "node_modules/font-awesome-sass/assets/stylesheets"
   },
   "scripts": {
-    "start": "concurrently \"npm run storybook:run\" \"npm run storybook:openurl\"",
+    "start": "concurrently \"yarn run storybook:run\" \"yarn run storybook:openurl\"",
```

Comment:
With yarn, you don't need to have the `run` command to execute the script, can just call it by its name `yarn storybook:run`

Reply:
You are right, I'll use the shortcut as suggested.

I tend to prefer an explicit `yarn run` since [CLI docs](https://yarnpkg.com/lang/en/docs/cli/run/) say:

> Note that built-in cli commands will have preference over your scripts, so you shouldn’t always rely on this shortcut in other scripts

but for stuff inside the `scripts` object, it should be fine.